src/common/configureStore.js

```diff
@@ -39,7 +40,7 @@ export default function configureStore({engine, initialState} = {}) {
   // }
 
   // Logger must be last middleware in chain.
-  if (process.env.NODE_ENV !== 'production') { // eslint-disable-line no-undef
+  if (process.env.NODE_ENV !== 'production' && process.env.IS_BROWSER) { // eslint-disable-line no-undef
```

Comment:
We shouldn't add logger when rendering on server, otherwise it fails because `window` is not defined.